src/lib/transaction.rs

```diff
@@ -76,7 +73,14 @@ impl Default for TransactionPayload {
             nullifiers: vec![],
             crossover: None,
             notes: vec![],
-            fee: Fee::default(),
+            fee: Fee {
```

Comment:
Don't we have a constructor? To directly set the internal attributes of the type is a dependency violation

Reply:
No. I already opened an issue to impl `Default` for PKI structures since it's a mess to workarround it.